src/ui/molecules/FileUpload.tsx

```diff
@@ -15,7 +15,7 @@ type FileStored = {
 };
 
 interface FileUploadContentProps {
-    status: 'IDLE' | 'UPLOADING' | 'COMPLETE' | 'ERROR';
+    status: 'IDLE' | 'UPLOADING' | 'COMPLETE' | 'ERROR' | 'PROCESSING';
```

Comment:
FileUpload ui component could use a unit test for this

Reply:
Previous tests were modified since the initial state changed. But just added a new test as well for progress 0